src/sentry/static/sentry/app/views/organizationEventsV2/transactionView/utils.tsx

```diff
@@ -236,11 +236,62 @@ export const getHumanDuration = (duration: number): string => {
   return `${durationMS.toFixed(3)} ms`;
 };
 
-const COLORS = ['#8B7FD7', '#F2BE7C', '#ffe066', '#74c0fc'];
-export const pickSpanBarColour = (spanNumberIndex: number) => {
-  spanNumberIndex = spanNumberIndex % COLORS.length;
-
-  return COLORS[spanNumberIndex];
+const COLORS = [
+  // row 1
+
+  // '#1E1F33',
+  // '#382947',
+  // '#522E4B',
+  // '#833054',
+  // '#AF2C41',
+  // '#98361B',
+  // '#C36609',
+  // '#967108',
+
+  // row 2
+
+  // '#444674',
+  // '#694E86',
+  // '#955389',
+  // '#C15584',
+  // '#E65D73',
+  // '#F27A58',
+  // '#F6983B',
+  // '#F2B712',
```

Comment:
Why is all this commented out?

Reply:
I was experimenting with few colour palettes 🎨